llvm/lib/Target/AArch64/MCTargetDesc/AArch64MCExpr.cpp

```diff
@@ -30,6 +30,7 @@ const AArch64MCExpr *AArch64MCExpr::create(const MCExpr *Expr, VariantKind Kind,
 }
 
 StringRef AArch64MCExpr::getVariantKindName() const {
+  // clang-format off
```

Comment:
We should probably let clang-format format this, does it make a total mess or is the diff just large?

Reply:
Allowing clang-format here leads to inconsistent formatting between new and old case labels - see a fragment below

```
  case VK_TLSDESC_PAGE:        return ":tlsdesc:";
  case VK_SECREL_LO12:         return ":secrel_lo12:";
  case VK_SECREL_HI12:         return ":secrel_hi12:";
  case VK_GOT_AUTH:
    return ":got_auth:";
  case VK_GOT_AUTH_PAGE:
    return ":got_auth:";
  case VK_GOT_AUTH_LO12:
    return ":got_auth_lo12:";
```

To make things consistent, I can format old cases the way clang-format likes (with return on new line). This should probably be done as a separate patch in order not to mess functionality and refactoring changes - if so, it's probably OK to leave `// clang-format off` until the follow-up refactoring patch merges. Please let me know your thoughts on this.

Reply:
I would just let clang-format do its thing. Depending on how bad the diff was, I might pre-commit it before the review. Doing it later is fine too though.

Reply:
OK, let's allow clang-format to do its job after the PR is merged. I'd prefer not to include large styling changes on unrelated code in this commit and do them separately to distinguish new functionality and style changes. It looks like it would be a bit easier to just delete these `// clang-format off` comments after merging the PR rather than generating a diff, pre-committing it now, resolving merge conflicts...

The same applies to the comment in AArch64MCExpr.h below.

Please let me know if you have some concerns with this.

Reply:
Sounds good